submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+  },
+];
+
+document.querySelector('.menu-list').addEventListener('click', function (e) {
+  if (e.target && e.target.nodeName === 'LI') {
+    switch (e.target.className) {
+      case 'menu-item-1':
+        showContent('menu-item-1');
+        break;
+      case 'menu-item-2':
+        showContent('menu-item-2');
+        break;
+      case 'menu-item-3':
+        showContent('menu-item-3');
+        break;
+      case 'menu-item-4':
+        showContent('menu-item-4');
+        break;
+      case 'menu-item-5':
+        showContent('menu-item-5');
+        break;
+    }
+  }
+});
+
+function showContent(nameClass) {
+  let element = document.querySelector('#content');
```

Comment:
If you do not change the value that is stored in a variable, it is better to use `const`